### PR TITLE
Update subgraph settings and tests

### PR DESCRIPTION
**Pull request by vderunov**

### CODE REVIEW

File: tests/.latest.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (noisekit) [inline tests/.latest.json]: -> .gitignore

and others
```ignore
.bin/
tests/.bin/
tests/.latest.json
subgraph.yaml
deployments/
build/
```

**Review by noisekit — APPROVED** [review]
1. add gitignore
2. fix revoke (need to have same-ish logic as when granting role)

Rest looks good

File: tests/.latest.json (hunk 1 above)
Comment (vderunov) [reply tests/.latest.json]: Updated  `.gitignore`  and removed unnecessary files.

**Review by noisekit — APPROVED** [review]

**Pull request merged**